base/pkg/entry.jl

```diff
@@ -51,7 +51,7 @@ function add(pkg::AbstractString, vers::VersionSet)
     @sync begin
         @async if !edit(Reqs.add,pkg,vers)
             ispath(pkg) || throw(PkgError("unknown package $pkg"))
-            info("Nothing to be done")
+            info("`$pkg` package is already installed")
```

Comment:
Mostly asking if it happens in situations where "is already installed" wouldn't make sense - like if it just got installed from the same call?